daprdocs/content/en/operations/support/support-versioning.md

```diff
@@ -0,0 +1,102 @@
+
+Samples and examples in SDKs version with that repo.
+
+## Components
+Components are implemented in a component-contrib repository and follow a `MAJOR` versioning scheme. The version for components adheres to major versions (vX), as patches and non-breaking changes are added to the latest major version. The version is incremented when there’s a non-backwards compatible change in a component interface, for example, changing an existing method in the State Store interface.  
+
+The [components-contrib](https://github.com/dapr/components-contrib/) repo release is a flat version across all components inside.  That is, a version for the components-contrib repo release is made up of all the schemas for the components inside it. A new version of Dapr does not mean there is a new release of components-contrib if there are no component changes. 
+
+Note: Components have a production usage lifecycle status: Alpha, Beta and GA (stable). These statuses are not related to their versioning. The tables of supported components shows both their versions and their status.
+* List of [state store components]({{< ref supported-state-stores.md >}})
+* List of [pub/sub components]({{< ref supported-pubsub.md >}})
+* List of [secret store components]({{< ref supported-secret-stores.md >}})
+* List of [binding components]({{< ref supported-bindings.md >}})
+
+### Component schemas 
+
+Versioning for component YAMLs comes in two forms:
+- Versioning for the component manifest. The `apiVersion`
+- Version for the component implementation. The `.spec.version`
+
+A component manifest includes the schema for an implementation in the `.spec.metadata` field, with the `.type` field denoting the implementation
+
+See the comments in the example below:
+```yaml
+apiVersion: dapr.io/v1alpha1 # <-- This is the version of the component manifest
+kind: Component
+metadata:
+  name: pubsub
+spec:
+  version: v1 # <-- This is the version of the pubsub.redis schema implementation
+  type: pubsub.redis
+  metadata:
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    value: general-kenobi
+```
+### Component manifest version
+The Component YAML manifest is versioned with `dapr.io/v1alpha1`. 
+
+### Component implementation version
+The version for a component implementation is determined by the `.spec.version` field as can be seen in the example above. The `.spec.version` field is mandatory in a schema instance and the component fails to load if this is not present. For the release of Dapr 1.0.0 all components are marked as `v1`.The component implementation version is incremented only for non-backward compatible changes.
+
+### Component deprecations
+Deprecations of components will be announced two (2) releases ahead. After 2 releases, the component is unregistered from the Dapr runtime, and trying to load it issues a warning.
```

Comment:
This is incorrect. If it is unregistered from the Dapr runtime, it will throw a fatal exception not a warning.

Reply:
Fixed to say fatal error